tests/test_assignee.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbug.models.assignee import AssigneeModel
+
+
+def test_get_assignee_labels():
+    # Due to the lack of data in fixtures/bugs.json, classes is always empty.
```

Comment:
We need to add the required data to the fixture DB, otherwise the test is useless.

Reply:
Yes, I think so too. But I don't know how to add more data. Where are these bug data came from in bugs.json?

Reply:
They come from data/bugs.json